sw/device/silicon_creator/rom/rom.c

```diff
@@ -330,14 +330,15 @@ static rom_error_t rom_verify(const manifest_t *manifest,
       &ecdsa_key));
   // SPX+ key.
   const sigverify_spx_key_t *spx_key = NULL;
+  const sigverify_spx_config_id_t *spx_config = NULL;
```

Comment:
Since this value is just an integer, we probably don't need a pointer to it here -- we can just get a copy of it from `spx_key_get` and pass-by-value to `spx_verify`.

If you're concerned with having an initial invalid value, zero or all-ones would be perfectly good as invalid values.